dashboard/cookbook/hubspotforms.md

```diff
@@ -0,0 +1,123 @@
+
+**Method #2** relies on HubSpot for localized forms and uses a proxy-compatible injected JS approach.
+
+## Method #1: Proxy
+
+The proxy approach traces he structure of the main and form domains via linked projects. Affected JS resources/endpoints are overridden and the responses marked as translatable.
+
+### Project Creation & Setup
+
+HubSpot uses several external domains to drive a form. You will see   [https://js.hsforms.net/forms/v2.js](https://js.hsforms.net/forms/v2.js) referenced in the page source. This file itself references [https://forms.hubspot.com](https://forms.hubspot.com), which is where the translatable form contents are coming from.
+
+Domains used by a HubSpot form are related to the main project and each other in the following manner:
+
+![HubSpot Projects](/img/dot-graphs/hs-form-w-js.png)
+
+Assuming that `example.com` is already set up, at most two additional projects are required:
+
+1. `js.hsforms.com`: creation of this project is *optional*, though not complicated. At the time of writing, visiting the landing page results in a `403 Forbidden` page (but this is not a problem).
+
+2. `forms.hubspot.com`: this URL redirects you to `https://developers.hubspot.com/`. To create a project for it, **disable Redirect checking** in the Add Project dialog. Click on **Advanced settings** to reveal the option and uncheck the checkbox:
+
+![Disabling Redirect checks](/img/dashboard/create_project_disable_redirect_check.png)
+
+Don't forget to add every target language of the main project to each project you create.
+
+### Link Projects
+
+Open each project in a separate tab and link each project according to the [section on Project Linking]("../../menu/dashboard/linkprojects.hhtml"). The result should be a chain of projects leading from `example.com` to `forms.hubspot.com` with `js.hsforms.net` as an intermediary.
+
+### Alternative: Search & Replace
+
+The `js.hsforms.net` project is not, strictly speaking, necessary. Its true purpose is merely to expose a slightly modified version of the `/forms/v2.js` script. If its URL is referred to in a way that makes it possible, you can *sidestep* the domain using a combination of [Search & Replace](../../menu/dashboard/pathsettings.html#search-replace-override) and a [page content override](../../menu/pagemodifiers/contentoverride.html). The setup steps for this are as follows (done on the main project):
+
+1. **create a path override** for the exact URL where the form is present (the diagram above shows `/contact` as an illustration).
+
+2. **add a regex Search & Replace rule**: replace `https?://js.hsforms.net` with the empty string to turn the reference to`/forms/v2.js` into a relative URL (and effectively point it toward a page content override on the project domain that will be created in a moment).
```

Comment:
@lsoos I think it would be nice to re-phrase this sentence in the way that is more apparent that the `https?://js.hsforms.net` string is not containing a question mark because of a type, but this is a regular expression.